<launch>
    <!-- This node will launch frame conversion from depth image (typically from Realsense D4xx camera) to mavros OBSTACLE_DISTANCE message -->
    <arg name="debug_enable" default="true"/>
    <param name="debug_enable"   value="$(arg debug_enable)" />
    
    <node pkg="vision_to_mavros" type="d4xx_to_mavros_node" name="d4xx_to_mavros" output="screen" >
        <remap from="depth_rect_image" to="/d435i/depth/image_rect_raw" />
    </node>
</launch>